Tests/KakapoDBTests.swift

```diff
@@ -229,37 +229,40 @@ class KakapoDBTests: QuickSpec {
             
             it("should not update different kind of objects from different databases with same id") {
                 let anotherDB = KakapoDB()
-                sut.create(UserFactory.self, number: 20)
-                anotherDB.create(CommentFactory.self, number: 20)
-                expect{ try sut.update(anotherDB.find(CommentFactory.self, id: 5)!) }.to(throwError(errorType: KakapoDBError.self))
-                let updatedCommentInDb = sut.find(CommentFactory.self, id: 5)
+                sut.create(UserFactory.self, number: 2)
+                anotherDB.create(CommentFactory.self, number: 2)
+                expect{ try sut.update(anotherDB.find(CommentFactory.self, id: 0)!) }.to(throwError(errorType: KakapoDBError.self))
+                let updatedCommentInDb = sut.find(CommentFactory.self, id: 0)
```

Comment:
because it will be faster when all test creates 2/3 object instead of 20 and doesn't change anything